lambdas/service/app.py

```diff
@@ -1557,19 +1558,23 @@ def file_to_drs(doc):
     """
     Converts an aggregate file document to a DRS data object response.
     """
-    replicas = ['aws']
+    urls = [
+        file_url(uuid=doc['uuid'],
+                 version=doc['version'],
+                 replica='aws',
```

Comment:
Given the intended use of this endpoint, and that are trying to get a `gs://` URL, I think the `replica` value here be changed from `aws` to `gcp`.